tests/integration/stack_reference/dotnet/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+using System.Threading.Tasks;
+using Pulumi;
+
+class Program
+{
+    static Task<int> Main(string[] args)
+    {
+        return Deployment.RunAsync(() => 
+        {
+            var config = new Config();
+            var org = config.Require("org");
+            var slug = $"{org}/{Deployment.Instance.ProjectName}/{Deployment.Instance.StackName}";
+            //TODO var a = new StackReference(slug);
```

Comment:
Do we support stack references yet?

Reply:
Not yet.  I will add to the list to have in the future.